contracts/L2TimelockTrigger.sol

```diff
@@ -48,39 +46,39 @@ contract L2TimelockTrigger is Manageable {
 
     /**
      * @notice Initialize L2TimelockTrigger smart contract.
-     * @param _owner                       Address of the L2TimelockTrigger owner.
-     * @param _prizeDistributionHistory PrizeDistributionHistory address
-     * @param _drawHistory                DrawHistory address
-     * @param _timelock           Elapsed seconds before new Draw is available
+     * @param _owner                   Address of the L2TimelockTrigger owner.
+     * @param _prizeDistributionBuffer PrizeDistributionBuffer address
+     * @param _drawBuffer              DrawBuffer address
+     * @param _timelock                Elapsed seconds before timelocked Draw is available
      */
     constructor(
         address _owner,
-        IDrawHistory _drawHistory,
-        IPrizeDistributionHistory _prizeDistributionHistory,
+        IDrawBuffer _drawBuffer,
+        IPrizeDistributionBuffer _prizeDistributionBuffer,
         IDrawCalculatorTimelock _timelock
     ) Ownable(_owner) {
-        drawHistory = _drawHistory;
-        prizeDistributionHistory = _prizeDistributionHistory;
+        drawBuffer = _drawBuffer;
+        prizeDistributionBuffer = _prizeDistributionBuffer;
         timelock = _timelock;
 
-        emit Deployed(_drawHistory, _prizeDistributionHistory, _timelock);
+        emit Deployed(_drawBuffer, _prizeDistributionBuffer, _timelock);
     }
 
     /* ============ External Functions ============ */
 
     /**
      * @notice Push Draw onto draws ring buffer history.
      * @dev    Restricts new draws by forcing a push timelock.
-     * @param _draw Draw
-     * @param _prizeDistribution PrizeDistribution
+     * @param _draw              Draw struct from IDrawBeacon
+     * @param _prizeDistribution PrizeDistribution struct frmo IPrizeDistributionBuffer
```

Comment:
Small typo here, otherwise it's all good.
```suggestion
     * @param _prizeDistribution PrizeDistribution struct from IPrizeDistributionBuffer
```